<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ma première UI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nomLabel">
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="prNomLabel">
       <property name="text">
        <string>prénom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prNomLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateDeNaissanceLabel">
       <property name="text">
        <string>date de naissance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dateDeNaissanceLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="villeLabel">
       <property name="text">
        <string>ville</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="villeLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>VALIDE BRO</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultText">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>229</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fic&amp;hier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
